cmd/internal/kube/kube.go

```diff
@@ -0,0 +1,15 @@
+package kube
+
+import (
+	"github.com/akitasoftware/akita-cli/cmd/internal/cmderr"
+	"github.com/pkg/errors"
+	"github.com/spf13/cobra"
+)
+
+var Cmd = &cobra.Command{
+	Use:   "kube",
+	Short: "Gateway to Kubernetes related utilities",
```

Comment:
```suggestion
	Short: "Install Akita in your Kubernetes cluster",
```

Let's try to make this action-oriented (why would they run this) rather than generic -- that's the pattern we used with 'akita ecs' and 'akita nginx'.